content/en/blog/_posts/Cluster-API-v1alpha3-Delivers-New-Features-and-an-Improved-User-Experience.md

```diff
@@ -0,0 +1,209 @@
+
+Following the v1alpha2 release in October 2019, many members of the Cluster API community met in San Francisco, California, to plan the next release. The project had just gone through a major transformation, delivering a new architecture that promised to make the project easier for users to adopt, and faster for the community to build. Over the course of those two days, we found our common goals: To implement the features critical to managing production clusters, to make its user experience more intuitive, and to make it a joy to develop.
+
+The v1alpha3 release of Cluster API brings significant features for anyone running Kubernetes in production and at scale. Among the highlights:
+* Declarative Control Plane Management
+* Support for Distributing Control Plane Nodes Across Failure Domains To Reduce Risk
+* Automated Replacement of Unhealthy Nodes
+* Support for Infrastructure-Managed Node Groups
+
+For anyone who wants to understand the API, or prizes a simple, but powerful, command-line interface, the new release brings:
+* Redesigned clusterctl, a command-line tool (and go library) for installing and managing the lifecycle of Cluster API
+* Extensive and up-to-date documentation in The Cluster API Book
+
+Finally, for anyone extending the Cluster API for their custom infrastructure or software needs:
+* New End-to-End (e2e) Test Framework
+* Documentation for integrating Cluster API into your cluster lifecycle stack
+
+All this was possible thanks to the hard work of many contributors.
+
+## Declarative Control Plane Management
+#### Special thanks to [Jason DeTiberus](https://github.com/detiber/), [Naadir Jeewa](https://github.com/randomvariable), and [Chuck Ha](https://github.com/chuckha)
+
+The Kubeadm-based Control Plane (KCP) provides a declarative API to deploy and scale the Kubernetes control plane, including etcd. This is the feature many Cluster API users have been waiting for! Until now, to deploy and scale up the control plane, users had to create specially-crafted Machine resources. To scale down the control plane, they had to manually remove members from the etcd cluster. KCP automates deployment, scaling, and upgrades.
+
+Here’s an example of a 3-replica control plane for the Cluster API Docker Infrastructure, which the project maintains for testing and development. For brevity, other required resources, like Cluster, and Infrastructure Template, referenced by its name and namespace, are not shown.
+
+```yaml
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+kind: KubeadmControlPlane
+metadata:
+  name: example
+spec:
+  infrastructureTemplate:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: DockerMachineTemplate
+    name: example
+    namespace: default
+  kubeadmConfigSpec:
+    clusterConfiguration:
+  replicas: 3
+  version: 1.16.3
+```
+
+Deploy this control plane with kubectl:
+```shell
+$ kubectl apply -f example-docker-control-plane.yaml
```

Comment:
nit: this isn't a valid shell script

Reply:
Right. This is a "literate" shell script. Including the prompt was how I thought I could distinguish input from output. Open to suggestions, though.

Reply:
For the main docs, the style guide suggests two blocks, something like:

> Run this command to list the changes on the branch:
> ```bash
> command git diff "$(git merge-base master HEAD)"..HEAD --name-only
> ```
> ```
> content/en/blog/_posts/Cluster-API-v1alpha3-Delivers-New-Features-and-an-Improved-User-Experience.md
> ```
>
> You can now use `cat` to read the article.

Reply:
Thanks, I'll use that.